fuse_models/include/fuse_models/odometry_2d_publisher.h

```diff
@@ -200,6 +201,11 @@ class Odometry2DPublisher : public fuse_core::AsyncPublisher
   std::unique_ptr<tf2_ros::TransformListener> tf_listener_;
 
   ros::Timer publish_timer_;
+
+  /**
+   * @brief A ros::console filter to print delayed throttle messages, that can be reset
+   */
+  std::unique_ptr<ros::console::FilterBase> delayed_throttle_filter_{ new fuse_core::DelayedThrottleFilter(10.0) };
```

Comment:
The `delayed_throttle_filter_` object is used within the `publish_timer_` callback. As such, it is safer to destroy the timer before destroying the throttle filter. Would you mind moving `delayed_throttle_filter_` above `publish_timer_` for that reason?

Reply:
Good catch. Moved.